tests/unit/Commands/ContinuousIntegration/TravisCi/TravisCiCommandTest.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+namespace QuickStrapUnit\Commands\ContinuousIntegration\TravisCi;
+
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use QuickStrap\Commands\ContinuousIntegration\TravisCi\TravisCiCommand;
+use QuickStrap\Commands\ContinuousIntegration\TravisCi\TravisCiQuestionProxy;
+use QuickStrap\Helpers\Composer\PackageHelper;
+use QuickStrap\Helpers\PathHelper;
+use Symfony\Component\Console\Helper\HelperSet;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class TravisCiCommandTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var  string */
+    private $configPath;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject */
+    private $questionHelper;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject */
+    private $packageHelper;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject */
+    private $pathHelper;
+    
+    /** @var  vfsStreamDirectory */
+    private $applicationPath;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject|TravisCiQuestionProxy */
+    private $questionProxy;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject|InputInterface */
+    private $input;
+    
+    /** @var  \PHPUnit_Framework_MockObject_MockObject|OutputInterface */
+    private $output;
+    
+    /** @var TravisCiCommand */
+    private $sut;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->applicationPath = vfsStream::setup('application_path');
+        $this->input = $this->getMock(InputInterface::class);
+        $this->output = $this->getMock(OutputInterface::class);
+        $this->questionHelper = $this->getMock(QuestionHelper::class);
+        $this->packageHelper = $this->getMock(PackageHelper::class);
+        $this->pathHelper = $this->getMock(PathHelper::class);
+        $this->questionProxy = $this->getMock(TravisCiQuestionProxy::class);
+
+        $this->configPath = sprintf('%s/.travis.yml', $this->applicationPath->url());
+        self::assertFileNotExists($this->configPath);
+        
+        $this->pathHelper->method('getPath')
+            ->willReturn($this->configPath);
+        
+        $this->sut = new TravisCiCommand('ci:travis-ci', $this->questionProxy);
+        $this->sut->setHelperSet(new HelperSet(array(
+            'question' => $this->questionHelper,
+            'package' => $this->packageHelper,
+            'path' => $this->pathHelper,
+        )));
+    }
+
+    /**
+     * @expectedException \Symfony\Component\Console\Exception\ExceptionInterface
+     */
+    public function testMissingPhpVersion()
```

Comment:
test missing php version what? Please make test names communicate the behavior they assert exists.